migrations/Version20220815191443.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DoctrineMigrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20220815191443 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return '';
+    }
+
+    public function up(Schema $schema): void
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->addSql('ALTER TABLE todo CHANGE is_completed is_completed TINYINT(1) DEFAULT 0 NOT NULL');
+    }
```

Comment:
Move it to parent migration

Reply:
[OK] Already at the latest version ("DoctrineMigrations\Version20220815191443")                                        
How can i do it?

Reply:
https://www.doctrine-project.org/projects/doctrine-migrations/en/3.5/index.html